### PR TITLE
Update horizon and rpc provider status

### PR DESCRIPTION
**Pull request by janewang**

### CODE REVIEW

File: docs/data/horizon/horizon-providers.mdx [hunk 1]
```diff
@@ -17,14 +17,14 @@ These providers allow access to the Testnet and Mainnet networks.
...
+| [QuickNode\*](https://www.quicknode.com/docs/stellar) | [Access](https://www.quicknode.com/chains/stellar) | Not available | Available | Available |
 | [Ankr](https://www.ankr.com/rpc/advanced-api/) | [Access](https://www.ankr.com/rpc/advanced-api/) | Not available | Available | Available |
-| [Validation Cloud](https://www.validationcloud.io/) | [Access](https://www.validationcloud.io/) | Not available | Coming Soon | Coming Soon |
+| [Validation Cloud\*](https://www.validationcloud.io/) | [Access](https://app.validationcloud.io/) | Not available | Available | Available |
 
-\*_Please note that Blockdaemon provides full historical data on its Horizon. Please note that Blockdaemon combines Horizon and RPC in the same endpoint._
+\*_Please note that Blockdaemon, Validation Cloud, and Quicknode combine Horizon and RPC in the same endpoint._
 
-\*\*_Please note that Quicknode combine Horizon and RPC in the same endpoint._
+†_Please note that Blockdaemon provides full historical data on its Horizon. Please note that Blockdaemon combines Horizon and RPC in the same endpoint._
```
Comment (ElliotFriend) [inline docs/data/horizon/horizon-providers.mdx:27]: I don't think we need to duplicate the bit about Blockdaemon combining things in the note after the `†` asterisk-thingy. Let's just leave that one at:
```suggestion
†_Please note that Blockdaemon provides full historical data on its Horizon._
```

**Review by ElliotFriend — COMMENTED** [review]
Thanks so much for keeping these up-to-date!

File: docs/data/horizon/horizon-providers.mdx (hunk 1 above)
Comment (janewang) [reply docs/data/horizon/horizon-providers.mdx:27]: Oh yes good point

Comment (janewang) [reply docs/data/horizon/horizon-providers.mdx]: Updated

**Review by ElliotFriend — APPROVED** [review]
looks great! Thanks!!

**Pull request merged**